lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -242,6 +244,81 @@ class DetailsRenderer {
     return tableElem;
   }
 
+  /**
+   * TODO(bckenny): migrate remaining table rendering to this function, then rename
+   * back to _renderTable and replace the original.
+   * @param {OpportunityDetails} details
+   * @return {Element}
+   */
+  _renderOpportunityTable(details) {
```

Comment:
this fn is a bit long, you foresee breaking it up in the future? add a TODO if so :)

Reply:
done!